docs/Collaboration guidelines.md

```diff
@@ -0,0 +1,51 @@
+Collaboration Guidelines
+===
+
+Be polite to each other
+---
+
+No one wants to wake up to a rude reply. Try to behave the best way you can, and show understanding when others are not. 
+
+When things escalate, try to use this document as a guideline on how to de-escalate them.
+
+Think of the project first
+---
+
+We're primarily here to make Reicast better.
+
+When discussing things in the tracker, forum or discord, try to think of the project first, and not of personal issues you might have with people.
+
+Good example: Constructive feedback is good because it makes contributors feel accepted, valued and safe. When people feel safe they are more comfortable explaining their opinion/viewpoint, and that helps make the project better
+
+Bad example: Emotionally abusing people is fun and makes my day better
+
+Avoid negativity, be realistic, give constructive feedback
+---
+
+Before replying, ask yourself the following questions.
+- How will the person I'm replying to feel after reading my comment?
+- Am I motivating them to get work done, or just spreading negativity?
+- Am I giving feedback that is valuable?
+
+Gracefully recover from missteps
+---
+
+We are all humans, and we have emotions and troubles, so often our conduct is not ideal. Here are some ideas on how to get things back on track.
+- When not thinking with a clear mind, take a break. Go for a walk, call a friend, stare at TV, smoke a cig, whatever works for you.
```

Comment:
Might not want to include smoking. You don't want some radical pointing out that you are motivating "bad habits" (even if that is all they contribute).